jetson/percep/pcl.cpp

```diff
@@ -316,8 +331,15 @@ bool PCL::CheckPath(const std::vector<std::vector<int>> &interest_points,
 
     bool end = true;
     
+    double previousDistance = DBL_MAX;
+
+    //if there are no interest points, the distance from the last obstacle should be DBL_MAX
+    distance = previousDistance;
```

Comment:
If DBL_MAX is a constant, is there a reason its not just assigned to distance in 337, rendering 334 unnecessary? And if it is not a constant, maybe the naming convention should be clarified

Reply:
`distance` is ultimately the variable that is returned. `previousDistance` and `currentDistance` are compared to each other to see whether `previousDistance` is less than the `currentDistance`, since we want to return the smallest distance. Technically, `distance` could be just `currentDistance`, but I used `currentDistance` to make it more clear (also `distance` is a variable in PCL)

Reply:
DBL_MAX is a constant value from the <float.h> class (from C++)